integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -486,6 +547,27 @@ public static Optional<URIStatus> waitForFileCompleted(FileSystem fileSystem, Al
     }
   }
 
+  /**
+   * Trys to lock the lock.
+   *
+   * @param lock the lock to lock
+   * @param message fail to lock message
+   * @param args fail to lock arguments
+   */
+  public static void tryLock(Lock lock, String message, Object... args) {
+    try {
+      if (!lock.tryLock(AlluxioFuseUtils.MAX_LOCK_WAIT_TIME,
+          TimeUnit.MILLISECONDS)) {
+        throw new DeadlineExceededRuntimeException(String.format(
```

Comment:
do we want to use the exception to indicate failure? 
tryLock should mean either a successful lock or a failure like busy or tryagain?

Reply:
This is used to share similar codes.
When tryLock failed, throw exception directly to the end user.
if locked, then should continue the operations